cargo-concordium/src/templates/cis2-nft-cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "cis2_wccd"
```

Comment:
Ideally this would be parametrized and the user would name it.

`cargo init` either takes the name of the directory you supply as the crate name, or if you don't supply a directory then it takes the name of the parent directory.

Could we do something similar?